ras/hwinfo.py

```diff
@@ -26,6 +26,9 @@ class Hwinfo(Test):
 
     is_fail = 0
 
+    def clear_dmesg(self):
+        process.run("dmesg -C ", sudo=True)
```

Comment:
@PraveenPenguin If there is no option in hwinfo to clear previous erros, then this is an acceptable way.